tools/azsdk-cli/Azure.Sdk.Tools.Cli/Models/Responses/CLICheckResponse.cs

```diff
@@ -0,0 +1,75 @@
+using System.Text.Json.Serialization;
+
+namespace Azure.Sdk.Tools.Cli.Models;
+
+/// <summary>
+/// Class for CLI check responses with exit code and output.
+/// </summary>
+public class ICLICheckResponse: Response
```

Comment:
I think we can remove this in favor of the process helper (though possibly move its process result class into models). Otherwise the naming needs to be changed as it's not an interface (no `I` prefix)